specification/monitor/resource-manager/microsoft.insights/2017-05-01-preview/diagnosticsSettings_API.json

```diff
@@ -0,0 +1,482 @@
+                  "eventHubAuthorizationRuleId":"/subscriptions/b67f7fec-69fc-4974-9099-a26bd6ffeda3/resourceGroups/andy1101/providers/Microsoft.EventHub/namespaces/andy1101/authorizationrules/RootManageSharedAccessKey",
+                  "workspaceId":"",
+                  "metrics": [
+                    {
+                      "category":"WorkflowMetrics",
+                      "enabled":true,
+                      "retentionPolicy": {
+                        "enabled":false,
+                        "days":0
+                      }
+                    }
+                  ],
+                  "logs": [
+                    {
+                      "category":"WorkflowRuntime",
+                      "enabled":true,
+                      "retentionPolicy": {
+                        "enabled":false,
+                        "days":0
+                      }
+                    }
+                  ]
+                }
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Gets the diagnostic setting": {
+            "$ref": "./examples/getDiagnosticSetting.json"
+          }
+        } 
+      },
+      "put": {
+        "tags": [
+          "DiagnosticSettings"
+        ],
+        "operationId": "DiagnosticSettings_CreateOrUpdate",
+        "description": "Creates or updates diagnostic settings for the specified resource.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceUriParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DiagnosticSettingsResource"
+            },
+            "description": "Parameters supplied to the operation."
+          },
+          {
+            "$ref": "#/parameters/NameParameter"
+          }
+        ],
+        "responses": {
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          },
+          "200": {
+            "description": "Successful request to create a diagnostic setting",
+            "schema": {
+              "$ref": "#/definitions/DiagnosticSettingsResource"
+            },
+            "examples": {
+              "application/json": {
+                "id":"/subscriptions/1a66ce04-b633-4a0b-b2bc-a912ec8986a6/resourcegroups/viruela1/providers/microsoft.logic/workflows/viruela6/diagnosticSettings/mysetting",
+                "type":"microsoft.logic/workflows",
+                "name":"mysetting",
+                "location":"eastus",
+                "tags":null,
+                "properties": {
```

Comment:
`tag` and `location` shouldn't be in the response according to `DiagnosticSettingsResource`